app/views/start/index.html.erb

```diff
@@ -1 +1,52 @@
-hej
\ No newline at end of file
+<!-- Header -->
+<div id="header">Header</div>
```

Comment:
Probably should add something more here, maybe just some kind of single-color navbar